saleor/plugins/webhook/tests/subscription_webhooks/fixtures.py

```diff
@@ -79,6 +79,132 @@ def subscription_category_deleted_webhook(subscription_webhook):
     )
 
 
+SHIPPING_PRICE_CREATED_SUBSCRIPTION_QUERY = """
+    subscription{
```

Comment:
Can you extend new tests, to fetch more fields than only `id`? This feature is still fresh, so would be good to be sure that we are able to provide more data than only `id`. 
Also, make sure that your branch contains changes from: https://github.com/saleor/saleor/pull/9520 as without this you can have a problem to fetch data.